scripts/install_git_crypt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eu
+
+_main() {
+  apk add --update --no-cache curl make g++ libressl libressl-dev
+  local tmpdir
+  tmpdir="$(mktemp -d git_crypt_install.XXXXXX)"
+
+  cd "$tmpdir"
+  curl -Lo git-crypt-0.6.0.tar.gz https://www.agwa.name/projects/git-crypt/downloads/git-crypt-0.6.0.tar.gz
+  tar -zxf git-crypt-0.6.0.tar.gz
+  cd git-crypt-0.6.0
+  make
+  make install
+  cd ..
+  rm -rf "$tmpdir"
+
+  apk del curl make libressl libressl-dev
```

Comment:
These packages are only useful for building git-crypt, but are not required at runtime